org.OpenT2T.Sample.SuperPopular.Thermostat/org.OpenT2T.Sample.SuperPopular.Thermostat.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE node PUBLIC "-//freedesktop//DTD D-BUS Object Introspection 1.0//EN" "http://standards.freedesktop.org/dbus/introspect-1.0.dtd"[]>
+<node>
+  <interface name="org.OpenT2T.Sample.SuperPopular.Thermostat">
+  
+    <!-- Turn on the HVAC -->
+    <method name="turnOn"/>
+    
+     <!-- Turn off the HVAC -->
+    <method name="turnOff"/>
+    
+    <!-- Get the current temperature -->
+    <method name="getCurrentTemperature">
+      <arg name="temp" type="d" direction="out" />
+    </method>
+   
+   <!-- set the current temperature -->
+    <method name="setCurrentTemperature">
```

Comment:
I think "get/setHeatSetpoint", "get/setCoolSetpoint" would be clearer because "setpoint" is the standard term for what this is doing. Reference the first sentence on the [Wikipedia Thermostat page](https://en.wikipedia.org/wiki/Thermostat).